test/inverse_of_test.rb

```diff
@@ -0,0 +1,53 @@
+require "test_helper"
+
+class InverseOfTest < IdentityCache::TestCase
+  def setup
+    super
+
+    @item = Item.create!(:title => 'bob')
+
+    AssociatedRecord.create!(:item => @item)
+    AssociatedRecord.create!(:item => @item)
+    AssociatedRecord.create!(:item => @item)
+  end
+
+  def test_fetch_should_setup_association_for_cache_belongs_to
+    Item.cache_belongs_to :associated
+
+    item = Item.find_by_id(@item.id)
+
+    record = item.fetch_associated
+    assert_equal item.object_id, record.item.object_id
+  end
+
+  def test_fetch_should_setup_association_for_cache_has_one
+    Item.cache_has_one :associated
+
+    item = Item.find_by_id(@item.id)
+
+    record = item.fetch_associated
+    assert_equal item.object_id, record.item.object_id
```

Comment:
This will always just delegate to the active record association, since item wasn't fetched from the cache and this is an embedded cached association.